SlevomatCodingStandard/Sniffs/ControlStructures/DisallowEqualOperatorsSniff.php

```diff
@@ -36,7 +36,8 @@ public function process(\PHP_CodeSniffer\Files\File $phpcsFile, $operatorPointer
 				$phpcsFile->fixer->endChangeset();
 			}
 		} else {
-			$fix = $phpcsFile->addFixableError('Operator != is disallowed, use !== instead.', $operatorPointer, self::CODE_DISALLOWED_NOT_EQUAL_OPERATOR);
+			$differentOperator = $tokens[$operatorPointer]['content'];
+			$fix = $phpcsFile->addFixableError('Operator ' . $differentOperator . ' is disallowed, use !== instead.', $operatorPointer, self::CODE_DISALLOWED_NOT_EQUAL_OPERATOR);
```

Comment:
Small cs suggestion: fewer variables, shorter lines:
```php
<?php
$fix = $phpcsFile->addFixableError(sprintf(
    'Operator %s is disallowed, use !== instead.',
    $tokens[$operatorPointer]['content']
), $operatorPointer, self::CODE_DISALLOWED_NOT_EQUAL_OPERATOR);
```

Reply:
Yes, sprintf is much better.

Reply:
Done :+1: